### ISSUE TITLE
May find this handy for adding type annotations

### ISSUE THREAD

**Issue by alok** [ipython/ipython#11774]
If you've seen it before, my apologies.

https://github.com/Instagram/MonkeyType

**Comment by Carreau** [comment]
Yes, I've seen it before; we are just using types annotations sparingly; 

Would be interesting to try in a couple of places see what it does.
